docs/developers/HowTo.md

```diff
@@ -54,7 +54,7 @@ ${GLUTEN_HOME}/dev/builddeps-veloxbe.sh --build_tests=ON --build_benchmarks=ON -
 
 ```
 cd ${GLUTEN_HOME}
-mvn clean package -Pspark-3.2 -Pbackends-velox -Pceleborn -Puniffle
+mvn clean package -Pspark-3.2 -Pbackends-velox -Pceleborn -Puniffle -DskipTests
 mvn test -Pspark-3.2 -Pbackends-velox -Pceleborn -pl backends-velox \
```

Comment:
Please help remove `-Pbackends-velox -Pceleborn` as they are not required here.

Reply:
@PHILO-HE  Thank you for the review. I was able to remove `-Pceleborn`, but if I remove `-Pbackends-velox`, I get the following error:
```
[ERROR] [ERROR] Could not find the selected project in the reactor: backends-velox @ 
```

Reply:
@dmsuehir, sorry for my wrong comment. `-Pbackends-velox` must be kept. We can remove `-Pceleborn -Puniffle`.

Reply:
Ok, thanks. I pushed another update. Let me know if that looks ok.